nexus/tests/integration_tests/utilization.rs

```diff
@@ -29,67 +40,128 @@ async fn test_utilization(cptestctx: &ControlPlaneTestContext) {
     );
 
-    assert_eq!(current_util[1].silo_name, "test-suite-silo");
-    assert_eq!(current_util[1].provisioned, SiloQuotasCreate::empty().into());
-    assert_eq!(
-        current_util[1].allocated,
-        SiloQuotasCreate::arbitrarily_high_default().into()
-    );
-
-    let _ = NexusRequest::object_put(
+    // in order to create resources in test-suite-silo, we have to create a user
+    // with the right perms so we have a user ID on hand to use in the authn_as
+    let silo_url = "/v1/system/silos/test-suite-silo";
+    let test_suite_silo: Silo = object_get(client, silo_url).await;
+    link_ip_pool(client, "default", &test_suite_silo.identity.id, true).await;
+    let user1 = create_local_user(
         client,
-        "/v1/system/silos/test-suite-silo/quotas",
-        Some(&params::SiloQuotasCreate::empty()),
+        &test_suite_silo,
+        &"user1".parse().unwrap(),
+        test_params::UserPassword::LoginDisallowed,
     )
-    .authn_as(AuthnMode::PrivilegedUser)
-    .execute()
     .await;
+    grant_iam(
+        client,
+        silo_url,
+        SiloRole::Collaborator,
+        user1.id,
+        AuthnMode::PrivilegedUser,
+    )
+    .await;
+
+    let test_project_name = "test-suite-project";
 
-    let current_util = objects_list_page_authz::<SiloUtilization>(
+    // Create project in test-suite-silo as the test user
+    NexusRequest::objects_post(
         client,
-        "/v1/system/utilization/silos",
+        "/v1/projects",
+        &params::ProjectCreate {
+            identity: IdentityMetadataCreateParams {
+                name: test_project_name.parse().unwrap(),
+                description: String::new(),
+            },
+        },
     )
+    .authn_as(AuthnMode::SiloUser(user1.id))
+    .execute()
     .await
-    .items;
+    .expect("failed to create project in test-suite-silo");
+
+    // Create instance in test-suite-silo as the test user
+    let instance_params = params::InstanceCreate {
+        identity: IdentityMetadataCreateParams {
+            name: "test-inst".parse().unwrap(),
+            description: "test instance in test-suite-silo".to_string(),
+        },
+        ncpus: InstanceCpuCount::try_from(2).unwrap(),
+        memory: ByteCount::from_gibibytes_u32(4),
+        hostname: "test-inst".parse().unwrap(),
+        user_data: vec![],
+        ssh_public_keys: None,
+        network_interfaces: params::InstanceNetworkInterfaceAttachment::Default,
+        external_ips: vec![],
+        disks: vec![],
+        boot_disk: None,
+        start: true,
+        auto_restart_policy: Default::default(),
+        anti_affinity_groups: Vec::new(),
+    };
+
+    NexusRequest::objects_post(
+        client,
+        &format!("/v1/instances?project={}", test_project_name),
+        &instance_params,
+    )
+    .authn_as(AuthnMode::SiloUser(user1.id))
+    .execute()
+    .await
+    .expect("failed to create instance in test-suite-silo");
```

Comment:
arguably this is kind of unnecessary since we do it with `default-silo` and the individual silo utilization response below, but I really wanted to exercise the list response